### PR TITLE
Added changes for 'Post' CRUD operations

### PR DESCRIPTION
**Pull request by arunselvakumar**

### CODE REVIEW

File: pages/api/posts/types.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export type Post = {
+  id: number;
+  title: string;
+  content: string;
```
Comment (swtshyl36) [inline pages/api/posts/types.ts:4]: rename to contentUrl

File: pages/api/posts/[id].ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import postService from './service';
+
+const handler = nc().get(async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const { id } = req.query;
+    const serviceResult = await postService.getPostById(+id);
+    return res.status(StatusCodes.OK).json(serviceResult);
+  } catch {
+    return res.status(StatusCodes.NOT_FOUND).end();
```
Comment (swtshyl36) [inline pages/api/posts/[id].ts:14]: We should not send NOT_FOUND error in case of exceptions. Pls Change logic accordingly

**Review by swtshyl36 — COMMENTED** [review]
Rename post to memes.

File: pages/api/posts/types.ts (hunk 1 above)
Comment (arunselvakumar) [reply pages/api/posts/types.ts]: We will have content for now.

File: pages/api/posts/[id].ts (hunk 2 above)
Comment (arunselvakumar) [reply pages/api/posts/[id].ts]: Fixed

**Review by swtshyl36 — APPROVED** [review]

**Pull request merged**